packages/frontend/src/components/EditorSettings/FlowTransfer.tsx

```diff
@@ -30,19 +32,44 @@ export function CustomSpinner() {
   )
 }
 
+const inputSchema = yup
```

Comment:
comment: ok for yup here since that's what the existing frontend code base uses. We sholuld file a ticket to move it to zod at some point.

Reply:
ok yep sounds good, thanks!